### PR TITLE
DevKit updates for 2.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -9,11 +9,8 @@ It's auto-generated by sonata-project/dev-kit package.
 <phpunit backupGlobals="false"
          backupStaticAttributes="false"
          colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
          failOnWarning="true"
-         processIsolation="false"
```
Comment (OskarStark) [inline phpunit.xml.dist:16]: Ping @franmomu

**Comment by OskarStark** [comment]
@franmomu can you please check Psalm here?

**Comment by franmomu** [comment]
@OskarStark I fixed the psalm issues in https://github.com/sonata-project/SonataTranslationBundle/pull/440

**Comment by OskarStark** [comment]
Let DevKit create a new one

**Pull request closed without merging**